HISTORY.rst

```diff
@@ -2,6 +2,21 @@
 History
 =======
 
+1.1.0 (2019-02-14)
+------------------
+
+* New ``is_included`` jinja variable which is a boolean indicating if the current file was included by
+  another ``devenv.yml`` file (``True``) or it is the original file passed to ``conda devenv`` (``False``) (`#72`_).
+
+* Added shortcuts to common jinja 2 checks, for example ``win`` which is equal to ``sys.platform.startswith("win")`` (`#75`_).
+
+* Added support conda-build-style YAML line comments (`` # [win]``) (`#79`_).
+
+.. _`72`: https://github.com/ESSS/conda-devenv/pull/72
```

Comment:
The linter doesn't like this... perhaps a `#` is missing before the number?

Reply:
Ops, fixed thanks